cmd/scaffold.go

```diff
@@ -130,7 +130,7 @@ func (fnb *FlowNodeBuilder) BaseFlags() {
 func (fnb *FlowNodeBuilder) EnqueueNetworkInit(ctx context.Context) {
 	fnb.Component("network", func(builder NodeBuilder, node *NodeConfig) (module.ReadyDoneAware, error) {
 
-		codec := jsoncodec.NewCodec()
+		codec := cborcodec.NewCodec()
```

Comment:
this change will also be needed here https://github.com/onflow/flow-go/blob/master/cmd/access/node_builder/access_node_builder.go#L644

Reply:
@vishalchangrani thanks for the comment!

Please see my response to @huitseeker below regarding testing. I'm wondering how I managed to run a 100 TPS load on localnet successfully for 600 seconds without the above change to `access_node_builder.go`? localnet does have an `access` node, so how could it have worked if it has a different codec?

Or is the link you kindly provided to the access node `JSON` configuration for something else, e.g. just for integration tests, or just for the access node communicating with the outside world, or something else not to do with network messages with the other nodes?

Reply:
ah! I can see where this confusion can arise from. So, the `EnqueueNetworkInit` here is not your regular Network init but the init of the `unstaked` network optionally used by the the access node if it supports an unstaked network. Currently, by default, the access node will not bring up this network.

However, things have changed since I made the comment 😄  and we will actually be removing/modifying this significantly since we decided we will not have two separate networks. So, you can either just make the change for consistency or leave it as-is.

Reply:
@vishalchangrani Oh, thank you very much for explaining how the confusion occurred.

I think I'd prefer to just leave it as-is because... I don't currently have a good way to test it unless there is a localnet option to bring up and use the `unstaked` network somehow? Or a different way to test it? And it sounds like that code is both not being currently used, and there will be no separate network in the future.

Reply:
Opened https://github.com/onflow/flow-go/issues/1147 to record this for follow-up.